worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/MultiRegionExtent.java

```diff
@@ -172,7 +172,7 @@ public IChunkSet processSet(IChunk chunk, IChunkGet get, IChunkSet set) {
             set = intersection.processSet(chunk, get, set);
         }
         if (disallowedIntersection != null) {
-            intersection.processSet(chunk, get, set);
+            set = intersection.processSet(chunk, get, set, true);
```

Comment:
shouldn't that be `disallowedIntersection` here?

Reply:
Yes